### PR TITLE
Make color_enabled consistent

### LINKED ISSUE #82: Color is not getting completely disabled
When disabling color, it seems that it does remove the escape codes from the diff, but not from the "Diff" header or the key.

### PR DESCRIPTION
**Pull request by tradiff**
Closes #82

Disabling color only applies to the diff, not the header and the key. Disabling color is needed for my team, because we use CircleCI which can show color sometimes, and remove it at other times (I believe based on the length of the log). And in the later case, we're seeing ansi color codes. I have confirmed the change proposed in #82 allows us to disable color in all of the output.

### CODE REVIEW

**Comment by tradiff** [comment]
@mcmire can we get this one merged too? 😄
Also BTW I noticed the gem hasn't been published from the previous PR merge.

**Comment by mcmire** [comment]
@travatomic Yeah. I think the only critique I have about this PR is that it'd be nice if we could add tests to ensure that this isn't an issue going forward. Let me look into how we could do this and get back to you.

File: support/test_plan.rb [hunk 1]
```diff
@@ -154,10 +154,6 @@ def run_test_with_libraries(*libraries)
       option_parser.parse!
     end
 
-    RSpec.configure do |config|
-      config.color_mode = color_enabled? ? :on : :off
-    end
```
Comment (tradiff) [inline support/test_plan.rb:161]: @mcmire have a look at this regarding the desire to test the bug fix. I think the reason the bug wasn't causing tests to fail in master is this line was copying the SuperDiff config to the RSpec config.

This was effectively hiding the fact that SuperDiff was inconsistently looking at both of these configs in two different places.

Removing this line makes the code in master fail specs, while the code in my PR here still passes all specs.

Comment (mcmire) [reply support/test_plan.rb:161]: Aha! Good find. Makes perfect sense!

**Comment by mcmire** [comment]
Something is goofy with CI. This seems to be a random failure — I don't think this has anything to do with this PR. I'll try to look at this so this can be merged quickly.

**Comment by mcmire** [comment]
Sorry it took me so long to merge this, thanks for the PR!

**Pull request merged**